src/data-browser/graphs/quarterly/index.jsx

```diff
@@ -37,8 +47,18 @@ export const QuarterlyGraphs = props => {
             history: props.history,
           }}
         />
-        <SectionFilerInfo show={section === 'Filer Info'} />
-        <SectionFAQ show={section === 'FAQ'} />
+
+        <Switch>
+          {/* Setting direct paths to access other tabs */}
+          <Route
+            path={"/data-browser/graphs/quarterly/info/filer"}
```

Comment:
How would you feel about the route being `/info/filers`, since the page presents info about multiple filers?
 
A section label of `Filer Info` still makes sense to me, I think it just sounds better than `Filers Info`.  Maybe the linguistically correct version would be `Filers' Info`?

Reply:
Any changes to this would change the tab label and update the URL. How would `Filers' Info` appear in the year? Look above for previous comment about this.

Reply:
As above, we can work to change the derivation of the URL from the label to be a mapped relationship. 
```
[{label: 'Graphs', url: '/graphs'}, {label: 'Filers Info', url: '/filers'}, {label: 'FAQ', url: '/faq'}]
```
 
 Then adjust the intermediary logic as needed.

Reply:
Updated code to use array of objects rather then array of strings to allow for more dynamic content control.